lms/djangoapps/instructor/views/api.py

```diff
@@ -885,6 +885,17 @@ def modify_access(request, course_id):
 
         # add CCX table entry
         if rolename == 'ccx_coach' and hasattr(course.id, 'ccx'):
+            existing_ccx_enrollement = CustomCourseForEdX.objects.filter(original_ccx_id=course.id.ccx, coach_id=user.id)
```

Comment:
Replace filter with count to save on resources. You are only checking if it exists, so it doesn't matter if it's an object or an integer.